recipes/qt5/bld.bat

```diff
@@ -16,15 +16,14 @@ IF "%DXSDK_DIR%" == "" (
     exit 1
 )
 
-curl -LO "http://download.qt.io/community_releases/%SHORT_VERSION%/%PKG_VERSION%/qtwebkit-opensource-src-%PKG_VERSION%.tar.xz"
-if errorlevel 1 exit 1
-7za x -so qtwebkit-opensource-src-%PKG_VERSION%.tar.xz | 7za x -si -aoa -ttar
-if errorlevel 1 exit 1
-MOVE qtwebkit-opensource-src-%PKG_VERSION% qtwebkit
-
 if "%DIRTY%" == "" (
     :: Shorten our build path as much as possible
     :: cd ..
+    curl -LO "http://download.qt.io/community_releases/%SHORT_VERSION%/%PKG_VERSION%/qtwebkit-opensource-src-%PKG_VERSION%.tar.xz"
+    if errorlevel 1 exit 1
+    7za x -so qtwebkit-opensource-src-%PKG_VERSION%.tar.xz | 7za x -si -aoa -ttar
+    if errorlevel 1 exit 1
+    MOVE qtwebkit-opensource-src-%PKG_VERSION% qtwebkit
```

Comment:
I do not agree with removing webkit.  Just because qtpy provides an abstraction layer doesn't mean everyone has one.  I think we need to keep webkit available to be consistent with the other VS versions.